deployment/terraform/strings.go

```diff
@@ -194,7 +194,6 @@ proxy_cache mattermost_cache;
 proxy_cache_revalidate on;
 proxy_cache_min_uses 2;
 proxy_cache_use_stale timeout;
-proxy_cache_lock on;
```

Comment:
It's fine to make the change here, but I'd also welcome a proper review of all these settings since I suspect most of us don't really know why and how they were decided.

Also pointing out we are documenting this as part of https://docs.mattermost.com/scale/high-availability-cluster.html